roles/stepup-azure-mfa/tasks/main.yml

```diff
@@ -34,9 +34,14 @@
 - name: Write Azure-MFA GSSP IdP private key
   copy: content="{{ azuremfa_idp_privatekey | vault(vault_keydir) }}" dest={{ component_config_file_dir_name }}/key.pem owner={{ component_owner }} mode={{ component_mode_400 }}
 
-
 # include ../../deploy/activate.yml
 - name: Activate Symfony component
   include_role:
     name: deploy
     tasks_from: activate
+
```

Comment:
Why did you choose to include the cleanup action for each component and not do this once from the deploy role?

Reply:
Because I only want to start cleaning up if activation went well